### DISCUSSION TITLE
📊  SP-API Health Dashboard in Seller Central Feedback

### DISCUSSION THREAD

**Discussion by sungolivia** [amzn/selling-partner-api-models#585] [Ideas]
Hey folks,

Not sure if you noticed in Seller Central, but we have health dashboard for SP-API now! 🚀 
https://sellercentral.amazon.com/sp-api-status 

It should look similar to the [MWS one](http://status.mws.amazon.com/) if anyone came from MWS world but the team is eager to hear any feedback you got for this health dashboard.

Let me know if:
- there anything else you are looking for?
- it is easy to navigate and use?
- you like it? 😄

**Comment by KyleJune** [comment]
It would be good to add a link to that status page from both the [SP-API documentation site](https://developer-docs.amazon.com/sp-api) and [Developer Central](https://sellercentral.amazon.com/sellingpartner/developerconsole). Developer central should have a link to the SP-API documentation site as well so those who are trying to develop an application can easily find the documentation for doing so.

**Reply by Akum777** [reply]
Noted. Thank you @KyleJune!

**Reply by chrislan815** [reply]
any update on this?

**Comment by informed-aaron** [comment]
Is there any way to subscribe to updates made to this new health dashboard? Even something basic like an RSS feed would allow integrators to be alerted to an incident instead of finding out from errors or from end users.

**Reply by sungolivia** [reply]
this is a great idea, let me inform the team about this one! Appreciate the feedback ❤️

**Comment by tomaszhanc** [comment]
@sungolivia not sure if the new dashboards work. On MWS we can see the outage of Feeds and Reports (which is happening right now):

<img width="759" alt="Screenshot 2022-11-23 at 07 51 53" src="https://user-images.githubusercontent.com/7013293/203487328-de818584-9049-4d7e-b49e-57374bc62178.png">

but on SP API everything is fine:

<img width="703" alt="Screenshot 2022-11-23 at 07 52 24" src="https://user-images.githubusercontent.com/7013293/203487418-402d06e6-e350-44ac-9b9a-75283a1f8a85.png">

<img width="717" alt="Screenshot 2022-11-23 at 07 52 31" src="https://user-images.githubusercontent.com/7013293/203487389-804a8c63-1c62-452e-b47b-a51d988aaf3a.png">

We are integrated with SP API for Feeds and Reports and the API returns 500 errors. Could you check it?

**Comment by jason1004** [comment]
Now a large number of getFeeds return a 500 status code error, but the new version of the health dashboard does not have any abnormalities.

![image](https://user-images.githubusercontent.com/13719529/203496208-623f4fe8-5eb4-4e94-87ab-ad714a0285fa.png)


![image](https://user-images.githubusercontent.com/13719529/203496240-79cb316a-8774-4be1-87a3-9ae2c2ed656c.png)


GET https://sellingpartnerapi-na.amazon.com/feeds/2021-06-30/feeds/154829019319 HTTP/1.1
Accept: application/json
X-Amz-Date: 20221123T074922Z
User-Agent: Swagger-Codegen/1.0.0/csharp
Host: sellingpartnerapi-na.amazon.com
Accept-Encoding: gzip, deflate


HTTP/1.1 500 Internal Server Error
Date: Wed, 23 Nov 2022 07:49:22 GMT
Content-Type: application/json
Content-Length: 158
Connection: keep-alive
x-amzn-RequestId: a78cb98d-c6de-4da4-8f6d-a07c4a9e8b0e
x-amz-apigw-id: cC2EaGhbIAMFTCQ=
X-Amzn-Trace-Id: Root=1-637dd082-732b42a01ec32b0c2d0d6833

{
  "errors": [
    {
      "code": "InternalFailure",
      "message": "We encountered an internal error. Please try again.",
      "details": ""
    }
  ]
}

**Comment by sungolivia** [comment]
hey everyone, thanks for reporting, our internal team is looking into this! Will provide an update as soon as I can.
